apps/server/src/signatures/dto/create-signature.dto.ts

```diff
@@ -1,6 +1,20 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { $Enums } from '@prisma/client';
-import { IsString, IsNotEmpty, IsNumber } from 'class-validator';
+import {
+  IsString,
+  IsNotEmpty,
+  IsNumber,
+  IsArray,
+  ValidateNested,
+} from 'class-validator';
+import { Type } from 'class-transformer';
+
+export class ConnectEmployeeDto {
```

Comment:
Create a new Dto to connect employee via id (because we can't create Employee objects in the frontend, but we can create a list of ids)